index.js

```diff
@@ -27,13 +27,19 @@ module.exports = {
     'stylelint-prettier',
   ],
   rules: {
+    // Declaring singular values of typography attributes (e.g. `font-weight`) is disallowed. Use a Backpack mixin that sets groups of attributes in sync with the overall design system
+    // https://github.com/Skyscanner/stylelint-plugin-backpack/blob/3acae45c28a015bbdd250b447803da1ac2ed31e8/lib/rules/use-typography-styles/index.js
     'backpack/use-typography-styles': [
       true,
       {
         severity: 'warning',
       },
     ],
-    'prettier/prettier': true,
```

Comment:
This was unnecessary.

This is already set by extending the config above:

```js
extends: ['stylelint-config-standard-scss', 'stylelint-prettier/recommended'],
```

https://github.com/prettier/stylelint-prettier/blob/main/recommended.js